### PR TITLE
tests(monitoring): Deflake monitoring sample tests

### PR DESCRIPTION
**Pull request by dazuma**
Monitoring sample tests would flake if multiple tests were run simultaneously, because the uptime check test would attempt to delete checks at the beginning, and that could stomp on checks being used by other concurrently running tests. Instead, we assume that tests clean up after themselves, and do not attempt to clean up at the beginning. We also provide a manual rake task that cleans the fixtures in a project if we should need it.

Also: added the proper "require" for minitest/hooks. Forgot to require it before, so the `before :all` and `after :all` were not behaving correctly.

### CODE REVIEW

**Comment by dazuma** [comment]
I think this should fix the test runs in #6497 and #6506.

File: google-cloud-monitoring/samples/Rakefile [hunk 1]
```diff
@@ -18,3 +18,16 @@ Rake::TestTask.new "test" do |t|
   t.test_files = FileList["acceptance/*_test.rb"]
   t.warning = false
 end
+
+task "clean_fixtures" do
```
Comment (quartzmo) [inline google-cloud-monitoring/samples/Rakefile:22]: I like `fixtures:clean` (using namespace) just in case we ever add any more fixtures tasks.

Comment (quartzmo) [reply google-cloud-monitoring/samples/Rakefile:22]: In KMS:

```
namespace :fixtures do
  desc "Lists the existing KMS acceptance test fixtures for the project in us-east1"
  task :list do
```

Comment (dazuma) [reply google-cloud-monitoring/samples/Rakefile]: Changed to `fixtures:clean`.

**Pull request merged**